index.html

```diff
@@ -0,0 +1,234 @@
+                </div>
+                <div class="connect">LET'S CONNECT</div>
+                <div class="link">
+                    <ul>
+                        <li>
+                            <a class="Github" href="">
+                                <img src="images/header/github.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="LinkedIn" href="">
+                                <img src="images/header/linkedin.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Ok" href="">
+                                <img src="images/header/ok.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="Twitter" href="">
+                                <img src="images/header/twiter.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="M" href="">
+                                <img src="images/header/m.png" alt="">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+               <button class="box">Get my resume</button>
+            </div>
+            <div id="box2">
+                <div class="lang">
+                    <h3>Languages</h3>
+                    <span class="material-symbols-outlined">
+                        expand_more
+                    </span>
+                </div>
+                <div class="languages">
+                    <div class="js">
+                        <img src="images/header/js.png" alt="js">
+                        <h3>JavaScript</h3>
+                    </div>
+                    <div class="html">
+                        <img src="images/header/html.png" alt="html">
+                        <h3>HTML</h3>
+                    </div>
+                    <div class="css">
+                        <img src="images/header/css.png" alt="css">
+                        <h3>CSS</h3>
+                    </div>
+                </div>
+                <div class="framework">
+                    <h1>Framework</h1>
+                    <span class="material-symbols-outlined">
+                        chevron_right
+                    </span>
+                </div>
+                <div class="line"></div>
+                <div class="skills">
+                    <h1>Skills</h1>
+                    <span class="material-symbols-outlined">
+                        chevron_right
+                    </span>
+                </div>
+                <div class="line"></div>
+            </div>
+        <!-- footer     -->
+        <footer>
+            <h2 class="lazy">Contact me</h2>
```

Comment:
- [ ] This section is not required for you to make it in this milestone thus it won't be reviewed yet 😄